ajax/libs/skeletonic/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "skeletonic",
+  "version": "1.0.89",
+  "description": "Skeletonic is a lightweight, intuitive and ultra-responsive CSS framework to speed up your Digital and Mobile Web development needs.",
+  "filename": "skeletonic.min-1.0.89.css",
```

Comment:
`filename` points to the main files of this library.
In the current file naming scheme, when a new version is released, we would need to update this field, that's why we hope files don't include version number.

Reply:
Version number has been removed on v1.0.93 at https://github.com/reedia/skeletonic/releases/tag/v1.0.93